src/main/resources/sql/municipality/get-all.sql

```diff
@@ -0,0 +1,3 @@
+SELECT *
+FROM municipality m
+INNER JOIN municipality_court mc ON mc.municipality_id = m.municipality_id
```

Comment:
Correct. There is no need to load them. It's all reference data anyway. It's not like judges which we always want embedded